Ginger/GingerCoreNET/Application Models/Execution/POM/POMExecutionUtils.cs

```diff
@@ -0,0 +1,55 @@
+﻿using amdocs.ginger.GingerCoreNET;
+using Amdocs.Ginger.Common.UIElement;
+using Amdocs.Ginger.Repository;
+using GingerCore.Actions;
+using GingerCore.Actions.Common;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Amdocs.Ginger.CoreNET.Application_Models.Execution.POM
+{
+    public class POMExecutionUtils
+    {
+        ActUIElement mAct = null;
+        public POMExecutionUtils(Act act)
+        {
+            mAct = (ActUIElement) act;
+        }
+
+        private string[] PomElementGUID => mAct.ElementLocateValue.ToString().Split('_');
+
+        public ApplicationPOMModel CurrentPOM { get { return GetCurrentPOM(); } }
```

Comment:
no need of this fields , directly make the methods public and will be called

Reply:
changes pushed..